schema/geometry_schema.json

```diff
@@ -623,6 +635,45 @@
 				"id",
 				"name"
 			]
+		},
+		"PitchedRoofType": {
+			"properties": {
+				"id": {
+					"type": "string"
+				},
+				"name": {
+					"type": [
+						"string",
+						"null"
+					]
+				},
+				"roof_type": {
+					"type": "string",
+					"enum": [
+						"gable",
```

Comment:
Let's upcase this enum {'Gable', 'Hip', 'Shed'}  changes needed in library, schema, and code; sorry :-(

@shorowit @bgschiller